ScriptingExample/src/test/java/org/openzen/zenscript/scriptingexample/tests/helpers/ZenCodeTestLoggerOutput.java

```diff
@@ -34,4 +34,14 @@ public void assertLineContains(int line, String content) {
 		final String foundLine = lines.get(line);
 		Assertions.assertTrue(foundLine.contains(content), "Expected this line to contain '" + content + "' but found '" + line + "'!");
 	}
+
+	public void assertLinesInOrder(String... lines) {
```

Comment:
💭 Discussion:  
What kind of helper functions to we want for that kind of stuff?
Also, should we just add [Hamcrest](https://hamcrest.org/JavaHamcrest/) or [AssertJ](https://assertj.github.io/doc/#assertj-core) and use their matchers?